packages/amplify-graphql-conversation-transformer/src/resolvers/invoke-lambda-resolver.ts

```diff
@@ -24,6 +26,14 @@ export const invokeLambdaMappingTemplate = (
   const RESPONSE_MUTATION_INPUT_TYPE_NAME = config.responseMutationInputTypeName;
   const MESSAGE_MODEL_NAME = config.messageModel.messageModel.name.value;
 
+  // TODO: Create and add these values to `ConversationDirectiveConfiguration` in an earlier step and
+  // access them here.
```

Comment:
Including this in a follow-up PR that consolidates the pipeline resolver generation logic.

Reply:
I assume this means that you want to derive the values closer to the ConversationDirective implementation, not that you're going to allow customers to configure these values?

Reply:
correct